gallery/src/styles/NxList/NxListClickableExample.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { faAngleRight } from '@fortawesome/free-solid-svg-icons';
+import { NxFontAwesomeIcon } from '@sonatype/react-shared-components';
+
+const NxListClickableExample = () =>
+  <ul className="nx-list nx-list--clickable">
+    <li className="nx-list__item">
+      <span className="nx-list__text">Action 1</span>
+      <NxFontAwesomeIcon icon={faAngleRight} className="nx-chevron" />
+    </li>
+    <li className="nx-list__item">
+      <span className="nx-list__text nx-truncate-ellipsis">
+        This list item should be truncated at the right end edge. youtube weathered network network systemic
+        systema claymore mine voodoo god garage monofilament realism order-flow corporation car footage vinyl.
+      </span>
+      <NxFontAwesomeIcon icon={faAngleRight} className="nx-chevron" />
```

Comment:
In this example the chevron is not vertically aligned with the text, and the list item is too tall for some reason.

Reply:
> In this example the chevron is not vertically aligned with the text, and the list item is too tall for some reason.

That is fixed now, probably as a part of: https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/b9593bb99ff1f7cc9785692b149281df1f97de79